pkg/kiosk/local_login.go

```diff
@@ -50,6 +51,11 @@ func GrafanaKioskLocal(cfg *Config, messages chan string) {
 	log.Printf("Sleeping %d MS before navigating to url", cfg.General.PageLoadDelayMS)
 	time.Sleep(time.Duration(cfg.General.PageLoadDelayMS) * time.Millisecond)
 
+	headers := make(map[string]interface{})
+	if len(cfg.BasicAuth.Username) != 0 && len(cfg.BasicAuth.Password) != 0 {
```

Comment:
same for this file, this is for local login (via ui interaction), and doesn't use basic auth, maybe this was not intended?

Reply:
see the following comment https://github.com/grafana/grafana-kiosk/pull/144#discussion_r1776464284
but if u think it's not needed then I'll just remove it.